src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs

```diff
@@ -39,8 +39,7 @@ public class NodeController : Controller
             _displayManager = displayManager;
             _factories = factories;
             _adminMenuService = adminMenuService;
-            _authorizationService = authorizationService;
-            New = shapeFactory;
+            _authorizationService = authorizationService;            
```

Comment:
Thanks @paragpkulkarni clean up is good.
Here, and for the others, I think you could remove them from the constructor as well, as if they are not being used, they do not need to be resolved.

Reply:
Removed IShapeFactory shapeFactory from constructor as per the comment